src/book.py

```diff
@@ -188,36 +188,95 @@ def _read_coinbase(self, file_path: Path) -> None:
 
                 # Unused variables.
                 del eur_total
-                del remark
 
```

Comment:
To keep the current format. Rename `_eur_spot` to `eur_spot` and add `del eur_spot`.